stdlib/unittest/mock.pyi

```diff
@@ -440,7 +440,7 @@ class _SpecState:
 
 def mock_open(mock: Any | None = ..., read_data: Any = ...) -> Any: ...
 
-PropertyMock = Any
+PropertyMock: TypeAlias = Any
```

Comment:
My reasoning was that we wanted type checkers to treat the class as if it were a type alias for `Any`. If we don't, I think we should _probably_ use `PropertyMock: Any` instead of `PropertyMock = Any`?

Reply:
Looking at the source code, though, you're right -- we can do better here.